library/src/main/kotlin/com/chuckerteam/chucker/api/ChuckerCollector.kt

```diff
@@ -74,23 +81,50 @@ public class ChuckerCollector @JvmOverloads constructor(
      * @param context Application context
      * @param startTimestamp The timestamp to read transactions from. Passing null means
      * transactions will not be limited by timestamp
+     * @param exportFormat The export format: LOG or HAR
      * @return The content uri of a file with the transactions in or null if the export failed.
      */
+    @SuppressLint("VisibleForTests")
```

Comment:
This needs to be removed before we can merge this.

Reply:
`@SuppressLint("VisibleForTests")` is there because HarUtils only exposes a suspend fun and i needed the underlying blocking method.
Not sure how to proceed here without changing the api or actually making the writeTransactions fun suspend.
I need some feedback here.

Reply:
You can make `HarUtils.fromHttpTransactions`  `internal` and remove all the `@VisibleForTesting` annotations and suppress